doc/source/tune/examples/tune-aim.ipynb

```diff
@@ -0,0 +1,198 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "ecad719c",
+   "metadata": {},
+   "source": [
+    "# Using Aim with Tune\n",
+    "\n",
+    "(tune-aim-ref)=\n",
+    "\n",
+    "[Aim](https://aimstack.io) is an easy-to-use & supercharged open-source experiment tracker.\n",
+    "Aim logs your training runs, enables a beautiful UI to compare them and an API to query them programmatically.\n",
+    "\n",
+    "```{image} /images/aim_logo_full.png\n",
+    ":align: center\n",
+    ":alt: Aim\n",
+    ":width: 100%\n",
+    ":target: https://aimstack.io\n",
+    "```\n",
+    "\n",
+    "Ray Tune currently offers built-in integration with Aim.\n",
+    "The {ref}`AimCallback <tune-aim-logger>` allows for automatic logging of metrics reported to Tune to the Aim API.\n",
```

Comment:
```suggestion
    "The {ref}`AimCallback <tune-aim-logger>` automatically logs metrics and reports them to Tune with the Aim API.\n",
```

Reply:
I apologize if I have changed the meaning of your sentence. Feel free to fix.

Reply:
Yes, it is actually the other way around. :) We report metrics to tune and aim logs them